lib/plugins/rest.js

```diff
@@ -43,7 +45,34 @@ exports.register = function (server, options, next) {
 				return awsAdapter.runInstances(instance);
 			})
 			.then(function (instance) {
+				var updatedInstance;
+
 				reply(instance).created("/v1/instances/" + instance.id);
+				awsInstanceID = instance.instanceID;
+				return sshAdapter.startSshPolling(awsInstanceID)
+				.then(function () {
+					return awsAdapter.getPublicIPAddress(awsInstanceID);
+				})
```

Comment:
I know we talked about this in person and I didn't have an answer at the time. Now that I see this code though, I wonder if we will ever want an instance and not want SSH enabled and the public IP available. This leads me to think that these function calls should be internal to the `AwsAdapter` and should be called implicitly before returning from `runInstances()`.